setup.cfg

```
@@ -1,28 +1,60 @@
+[metadata]
+name = omnifit
+# version should be PEP440 compatible (https://www.python.org/dev/peps/pep-0440/)
+version = 0.3
+author = Aleksi Suutarinen
+author_email = aleksi.suutarinen@iki.fi
+description = Omnifit - astronomical ice spectroscopy fitting
+long_description = Astronomical spectroscopy fitting and analysis of interstellar ices
+license = BSD 3-Clause
+url = https://github.com/RiceMunk/omnifit
+edit_on_github = False
+github_project = RiceMunk/omnifit
+python_requires = ">=3.6"
+
+[options]
+# install_requires should be formatted as a semicolon-separated list, e.g.:
+# install_requires = astropy; scipy; matplotlib
+install_requires = astropy; numpy; scipy; matplotlib; lmfit
```

```suggestion
install_requires = astropy>=4.0; numpy>=1.13; scipy; matplotlib; lmfit
```
I _think_ 1.13 is the minimum version to handle Quantities the way required now; you may also consider 1.16 (minimum for Astropy 4.0) or 1.18 (current minimum required by Astropy 5.0/5.1).

---

setup.cfg fixed in 7eaa9ae